webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala

```diff
@@ -35,6 +36,14 @@ import org.knora.webapi.util._
 
 import scala.concurrent.{ExecutionContext, Future}
 
+private case class validateUUIDOfResourceIRI(iri: IRI) {
```

Comment:
class name should be upper case

Reply:
Obsolete, I did some refactoring just before a while.